src/utilities/manageLikes.js

```diff
@@ -0,0 +1,34 @@
+import { storageAvailable } from "./testStorage";
+
+import fullHearth from "../assets/favorite/heart-full.svg";
+import emptyHearth from "../assets/favorite/heart-empty.svg";
+
+export const manageLikes = (event, fav) => {
+
+    const imgID = Number(event.target.dataset.id);
+
+    if (!storageAvailable("localStorage")) {
+        console.log("no access to localStorage");
+        return;
+    };
+
```

Comment:
I would put the constant after the condition. why declaring it if you may never use them ;-)

```suggestion
   
    if (!storageAvailable("localStorage")) {
        console.log("no access to localStorage");
        return;
    };

     const imgID = Number(event.target.dataset.id);

```

Reply:
yes, 🤦‍♂️ ... ressource management failed, thanks 😂